test.sh

```diff
@@ -13,4 +13,4 @@ export DAPP_LINK_TEST_LIBRARIES=1
 
 if [ ${1} ]; then match=${1}; dapp_test_verbosity=2; else match="contracts/test"; dapp_test_verbosity=1; fi
 
-LANG=C.UTF-8 dapp test --match "$match" --rpc-url "$ETH_RPC_URL" --verbosity $dapp_test_verbosity --cache "cache/dapp-cache" --fuzz-runs 100
+LANG=C.UTF-8 dapp test --match "$match" --rpc-url "$ETH_RPC_URL" --verbosity $dapp_test_verbosity --cache "cache/dapp-cache" --fuzz-runs 10
```

Comment:
do we want to bump up that to 10 again

Reply:
Yeah I want to personally - its starting to take a while locally